assets/scripts/menus/ContactMenu.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Menu from './Menu'
+// import { t } from '../app/locale'
+
+export default class ContactMenu extends React.PureComponent {
+  render () {
+    return (
+      <Menu name='contact' {...this.props}>
+        <a href='http://forums.streetmix.net/' target='_blank'>
+          <svg className='icon'>
+            <use xlinkHref='#icon-forums' />
+          </svg>
+          <span data-i18n='menu.contact.forums'>Discuss on the forums</span>
```

Comment:
I didn't really need to. The old way (using `data-i18n`' attributes) works just fine because these elements are always on the page and are translated in a brute-force sort of way ((1) search for all elements with `data-i18n` attributes; (2) replace text) whenever the language setting changes. The `t()` function was necessary for components that were dynamically generated so it needed to retrieve text based on the current locale.

Since `t()` would need to be called for every text string whenever a component `render()`s, leaving some text as `data-i18n` triggers (which is set-it-and-forget-it) seemed more performant. This isn't really ideal for consistency purposes, but I thought I'd kick this decision-making into the future. I've been trying to absorb [React-Intl](https://github.com/yahoo/react-intl) as an alternative, mostly because I like the idea of outsourcing libraries to very large companies.